src/database/postgres.ts

```diff
@@ -338,30 +340,40 @@ export class DatabaseWrapperPostgres implements DatabaseWrapper {
 		});
 	}
 
-	async registerBye(tournamentId: string, playerId: string): Promise<void> {
+	/// @throws {TournamentNotFoundError}
+	/// @throws {AssertStatusError}
+	/// @throws {EntityNotFoundError}
+	private async byeHelper(
```

Comment:
This pattern for saving repeated code over parallel operations is nifty, but do we do similar with add/remove hosts and channels? Seems inconsistent.

Reply:
If there's any there I can do it.

Reply:
Ah, there's less repetition in those because we're modifying an array rather than toggling a bool. Never mind.